yt_dlp/extractor/la7.py

```diff
@@ -58,12 +60,15 @@ def _generate_mp4_url(self, quality, m3u8_formats):
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
 
-        if not url.startswith('http'):
-            url = '%s//%s' % (self.http_scheme(), url)
+        if self._search_regex(
+                r'(drmsupport\s*:\s*true)\s*', webpage, video_id,
+                flags=re.IGNORECASE, fatal=False):
```

Comment:
```suggestion
        if self._search_regex(
                r'(?i)(drmsupport\s*:\s*true)\s*', webpage, video_id, default=None):
```
or even just re.search is fine here
```suggestion
        if re.search(r'(?i)(drmsupport\s*:\s*true)\s*', webpage):
```